static/js/common/init.js

```diff
@@ -95,8 +95,8 @@ MEDUSA.common.init = function() {
             prefixFromNow: 'In ',
             suffixAgo: 'ago',
             suffixFromNow: '',
-            seconds: 'less than a minute',
-            minute: 'about a minute',
+            seconds: 'few seconds',
```

Comment:
this should be "a few seconds" otherwise you'll get "few seconds ago" or "in few seconds".